### ISSUE TITLE
Fix looped tests

### ISSUE THREAD

**Issue by slaskawi** [project-ncl/pnc#142]
Examples:
* http://10.3.10.33:8081/job/NCL_Pull_Requests/232/console 
* http://10.3.10.33:8081/job/NCL_Pull_Requests/229/console 
* http://10.3.10.33:8081/view/4_JDG_6_5_x/job/NCL_Pull_Requests_wildfly/102/console

**Comment by jdcasey** [comment]
I think I have it fixed but I need to run a release of aprox, then update the demo vm before I can send a PR. ...and I'm out sick today. I'll resume work on this on Monday.

**Comment by jdcasey** [comment]
Okay, I ran into this again while trying to release aprox itself. For posterity, I filed/closed: https://github.com/jdcasey/aprox/issues/94 and I believe I have it fixed now. I ran the aprox build something like 30 times with all integration tests running (each build will run three or four test suites about 6-9 times each under different conditions, and they all use the client api)...after correcting the issue, the builds all completed successfully.

I don't think this was really a problem of conflicting with another process or anything, it was just luck of the draw, and running a LOT of executions would show the problem. As in, running between 18 and 36 test suites (I don't have the concrete count) wasn't showing it. Something about the PNC build seemed to make it more likely, perhaps heavier usage of the client APIs per test.

I'm releasing now, and should be ready to deploy to the demo VM and create a PR for this tomorrow.
